src/main/scala/org/broadinstitute/dsde/firecloud/service/EntityService.scala

```diff
@@ -33,10 +33,45 @@ trait EntityService extends HttpService with FireCloudDirectives {
   private final val ApiPrefix = "entities"
   private implicit val executionContext = actorRefFactory.dispatcher
 
-  val routes = listEntitiesPerTypeRoute
+  val routes = listEntityTypesRoute ~ listEntitiesPerTypeRoute
 
   lazy val log = LoggerFactory.getLogger(getClass)
 
+  @Path("")
```

Comment:
You can lose this line, the path is inherited from the class level `Api` annotation value property.